applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -205,3 +205,14 @@ The |usb_state| is a transport for :ref:`nrf_desktop_config_channel` when the ch
 
 The module also handles a HID keyboard LED output report received through USB from the connected host.
 The module sends the report using :c:struct:`hid_report_event`, that is handled either by :ref:`nrf_desktop_hid_state` (for peripheral) or by the :ref:`nrf_desktop_hid_forward` (for dongle).
+
+nRF54H20 support
+================
+
+The only supported USB stack for the nRF54H20 is the USB next stack.
+Due to the nRF54H20 USB driver characteristics, following changes have been made in the USB state module to support the nRF54H20 platform:
```

Comment:
`USB driver` -> `USB device controller (UDC)` ? (to be more consistent with Zephyr codebase)